src/gui/qgsnewgeopackagelayerdialog.cpp

```diff
@@ -70,18 +70,18 @@ QgsNewGeoPackageLayerDialog::QgsNewGeoPackageLayerDialog( QWidget *parent, Qt::W
   mGeometryTypeBox->addItem( tr( "Point" ), wkbPoint );
   mGeometryTypeBox->addItem( tr( "Line" ), wkbLineString );
   mGeometryTypeBox->addItem( tr( "Polygon" ), wkbPolygon );
-  mGeometryTypeBox->addItem( tr( "Multi point" ), wkbMultiPoint );
-  mGeometryTypeBox->addItem( tr( "Multi line" ), wkbMultiLineString );
-  mGeometryTypeBox->addItem( tr( "Multi polygon" ), wkbMultiPolygon );
+  mGeometryTypeBox->addItem( tr( "MultiPoint" ), wkbMultiPoint );
+  mGeometryTypeBox->addItem( tr( "MultiLine" ), wkbMultiLineString );
+  mGeometryTypeBox->addItem( tr( "MultiPolygon" ), wkbMultiPolygon );
 
 #if 0
   // QGIS always create CompoundCurve and there's no real interest of having just CircularString. CompoundCurve are more useful
   mGeometryTypeBox->addItem( tr( "Circular string" ), wkbCircularString );
 #endif
   mGeometryTypeBox->addItem( tr( "Compound curve" ), wkbCompoundCurve );
```

Comment:
Should this be CompoundCurve instead? (same for CurvePolygon, CircularString, etc.)